pom.xml

```diff
@@ -19,19 +19,26 @@
             <artifactId>selenium-java</artifactId>
             <version>4.9.1</version>
         </dependency>
-
```

Comment:
UTF encoding is in xml structure (1st line), this is redundant: "<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>"

Reply:
i can't understand this :(